### DISCUSSION TITLE
Get current session on a custom page

### DISCUSSION THREAD

**Discussion by limegorilla** [keystonejs/keystone#7917] [Questions]
Is there any way I can access the current session inside a custom page? I'm trying to create a dashboard and I'd need to be able to access the current user to be able to see their roles to ensure I am only showing information I should be showing them.

**Comment by flexdinesh** [comment]
You could use the generated query `authenticatedItem` to get current user's session data.

https://keystonejs.com/docs/apis/auth#graphql-api
